src/chainstate/stacks/db/mod.rs

```diff
@@ -595,17 +597,15 @@ pub const MINER_FEE_WINDOW: u64 = 24; // number of blocks (B) used to smooth ove
 // fraction (out of 100) of the coinbase a user will receive for reporting a microblock stream fork
 pub const POISON_MICROBLOCK_COMMISSION_FRACTION: u128 = 5;
 
-#[derive(Debug, Clone, Deserialize)]
-pub struct VestingSchedule {
-    pub address: StacksAddress,
+pub struct ChainStateAccountBalance {
+    pub address: String,
     pub amount: u64,
-    pub block_height: u64,
 }
 
-#[derive(Debug, Clone, Deserialize)]
-pub struct AccountBalance {
-    pub address: StacksAddress,
+pub struct ChainStateAccountVesting {
```

Comment:
Let's avoid calling this `vesting` -- @jessesoslow had a problem with this in Stacks 1.0.  The tokens aren't vesting in a legal sense -- they're already purchased and owned, but just not unlocked yet.

Reply:
This data is almost 1-1 from the Stacks 1.0 `account_vesting` table. Should it be renamed to `locked`?

Reply:
Sounds good to me -- we can call it `ChainstateAccountLockup` or something (also, `Chainstate` is one word).

Reply:
Done! Replaced the other `vesting` wording with `lockups`.